src/lib/types.ts

```diff
@@ -386,6 +386,8 @@ export type SignInWithPasswordCredentials =
         data?: object
         /** Verification token received when the user completes the captcha on the site. */
         captchaToken?: string
+        /** Messaging channel to use (e.g. whatsapp or sms) */
+        channel?: string
```

Comment:
Can we use `channel?: 'sms' | 'whatsapp'` instead?

Reply:
I would love for some docs on the method itself saying that if you want to use WhatsApp you need to configure it first.

Reply:
I will also follow up with a docs PR describing setup on supabase/supabase